### ISSUE TITLE
Lift feature

### ISSUE THREAD

**Issue by DITTY-1** [cfloutier/k2d2#18]
Hello, I am trying to use the lift feature. Maybe I am missing something, but when I hit start, and commence the launch, nothing happens. Am I correct in my understanding that the mod should be keeping my vehicle in an optimal ascent trajectory? I see the mod is active as it is updating, but my vehicle course is not being affected at all.

Is this a bug, or am I just using the mod incorrectly? I installed it via CKAN.

**Comment by cfloutier** [comment]
Strange..... The pilot just correct the direction of the ship using the SAS if the SAS is not active it can be the cause of your trouble

@DITTY-1 please let me know if the latest version works better

**Comment by kfsone** [comment]
It works fine for me, the things I've found:

1- The staging feature went away in the last release,
2- The profile is different than you might be expecting from MechJeb - the default profile doesn't start the turn until 2km (thanks @cfloutier, that has saved several of my vehicles :)
3- Once you reach the "final" altitude, if you have Maneuver Node Controller it will pause the game and give you a chance to throw a circularization burn in at the top,

#3 is a really epic feature, but it's never gotten me a circular orbit yet - actually it always leaves me with a PE under 70k from Kerbin - I tend to end up on my space-isp motors (eg poodle) for the circularization.

Nice work, @cfloutier, this was the droid I was looking for :)

**Comment by Guedoj** [comment]
Will auto-staging be coming back?

**Comment by cfloutier** [comment]
Yes soon, I just had a bug in latest version and I would like to release it sooner. It will be there in next

**Comment by Guedoj** [comment]
good to hear.
